### PR TITLE
Add seq::Renderer::requestRedraw()

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: seq/detail/masterConfig.cpp [hunk 1]
```diff
@@ -144,56 +144,61 @@ bool MasterConfig::handleEvent( const eq::ConfigEvent* event )
...
+        LBVERB << "Redraw: window change" << std::endl;
+        break;
+
+    case EVENT_REDRAW:
+        _redraw = true;
+        LBINFO << "Redraw request" << std::endl;
+        break;
+
+    default:
+        break;
```
Comment (eile) [inline seq/detail/masterConfig.cpp]: oh to the gods of merge conflicts! ;)

File: seq/detail/masterConfig.cpp [hunk 2]
```diff
@@ -144,56 +144,61 @@ bool MasterConfig::handleEvent( const eq::ConfigEvent* event )
...
+    case eq::Event::WINDOW_RESIZE:
+    case eq::Event::WINDOW_CLOSE:
+    case eq::Event::VIEW_RESIZE:
+        _redraw = true;
+        LBVERB << "Redraw: window change" << std::endl;
+        break;
+
+    case EVENT_REDRAW:
+        _redraw = true;
+        LBINFO << "Redraw request" << std::endl;
```
Comment (eile) [inline seq/detail/masterConfig.cpp]: LBINFO?

**Comment by eile** [comment]
+1

File: seq/detail/masterConfig.cpp (hunk 1 above)
Comment (tribal-tec) [inline seq/detail/masterConfig.cpp]: Haha :)

File: seq/detail/masterConfig.cpp (hunk 2 above)
Comment (tribal-tec) [inline seq/detail/masterConfig.cpp]: Oops. Happened when realizing I was in the wrong event handling ifdef path. When is the event cleanup merged? *poke*

**Pull request merged**